swan-lake/development-tutorials/ballerina-central/configure-proxy.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: ballerina-publishing-to-central-left-nav-pages-swanlake
+title: Configure proxy
+description: Below section describes how to configure proxy for accessing Ballerina Central, or custom repositories
+keywords: ballerina, programming language, configuring proxy, proxy, set-up proxy
+permalink: /learn/configure-proxy/
+active: configure-proxy
+intro: Below section describes how to configure proxy for accessing Ballerina Central, or custom repositories
+---
+
+## Configure proxy
+
+If you are connected to the internet via an HTTP proxy, you need to configure it in the `Settings.toml` file to carry out the Ballerina Central related operation. Add the following section to `Settings.toml` and change accordingly.
```

Comment:
```
If you are connected to the internet via an HTTP proxy, you need to configure it in the `Settings.toml` file to carry out the Ballerina Central related operation. 
```

Here, it would be better if you can explain what central related operations are.